src/structures/Widget.js

```diff
@@ -17,6 +17,14 @@ class Widget extends Base {
     this._patch(data);
   }
 
+  /**
+   * Represents a channel in a Widget
+   * @typedef {Object} WidgetChannel
+   * @property {Snowflake} [id] Id of the channel
+   * @property {string} [name] Name of the channel
+   * @property {number} [position] Position of the channel
```

Comment:
Any reason they're all possibly undefined?

Reply:
My bad I forgot how to to it x)
I'll update it